homeassistant/components/anthropic/__init__.py

```diff
@@ -20,7 +20,9 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: AnthropicConfigEntry) -> bool:
     """Set up Anthropic from a config entry."""
-    client = anthropic.AsyncAnthropic(api_key=entry.data[CONF_API_KEY])
+    client = await hass.loop.run_in_executor(
```

Comment:
We should use hass.async_add_executor_job so it gets tracked.

Also you could use a partial instead of a lambda